lib/astronoby/aberration.rb

```diff
@@ -22,27 +22,22 @@ def apply
       delta_longitude = Astronoby::Angle.as_degrees(
         -20.5 *
         Math.cos(
-          @sun_longitude.to_radians.value - @coordinates.longitude.to_radians.value
-        ) / Math.cos(@coordinates.latitude.to_radians.value) / 3600
+          @sun_longitude.radians - @coordinates.longitude.radians
+        ) / Math.cos(@coordinates.latitude.radians) / 3600
```

Comment:
Out of scope for this PR but for the future, I wonder if it would be useful to implement some math operators directly on `Angle`? Then you don't need to unwrap the angle values. Ideally you get to treat `Angle` instances like numbers, not just wrappers of numbers.

With `Angle#cos` and `Angle#-` this method would look like:

```ruby
(@sun_longitude - @coordinates.longitude).cos / @coordinates.lattitude.cos / 3600
````